fastlane/Fastfile

```diff
@@ -9,14 +9,26 @@ USER_ENV_FILE_PATH = File.join(Dir.home, '.simplenoteandroid-env.default')
 PROTOTYPE_BUILD_DOMAIN = 'https://d2twmm2nzpx3bg.cloudfront.net'
 
 before_all do
-  # Check that the env file exists
+  # TODO: Use EnvManager — https://github.com/wordpress-mobile/release-toolkit/pull/578
   UI.user_error!("#{USER_ENV_FILE_PATH} not found: Please copy env.example to #{USER_ENV_FILE_PATH} and fill in the values") unless is_ci || File.file?(USER_ENV_FILE_PATH)
+  Dotenv.load(USER_ENV_FILE_PATH)
 end
 
-# Constants
 PROJECT_ROOT_FOLDER = File.dirname(File.expand_path(__dir__))
-GITHUB_REPO = 'automattic/simplenote-android'
+
+ORGANIZATION = 'automattic'
+REPO_NAME = 'simplenote-android'
+GITHUB_REPO = "#{ORGANIZATION}/#{REPO_NAME}".freeze
 DEFAULT_BRANCH = 'trunk'
+BUILDKITE_ORG = ORGANIZATION
+BUILDKITE_PIPELINE = REPO_NAME
```

Comment:
The idea with these over-engineered constants is that they should be easy to copy paste into other `Fastfile`s.

The more homogeneous we can keep the `Fastfile`s the easier it should be to maintain them.